docs/TuxActionStores.md

```diff
@@ -0,0 +1,179 @@
+#Tux ActionStores
+##Table of Contents
+<ol>
+  <li><a href="#Context">Context</a></li>
+  <li><a href="#Premise">Premise</a></li>
+  <li><a href="#Implementation">Implementation</a></li>
+    <ol style="list-style-type:upper-alpha">
+      <li><a href="#Requiring-ActionStores">Requiring ActionStores</a></li>
+      <li><a href="#Creating-ActionStores">Creating ActionStores</a>
+        <ol style="list-style-type:upper-roman">
+          <li><a href="#ActionStores-createStore">METHOD: ActionStores.createStore</a></li>
+            <ol style="list-style-type:lower-alpha">
+              <li><a href="#storeProps-register">OBJECT: storeProps</a>
+                <ol style="list-style-type:lower-roman">
+                  <li><a href="#storeProps-register">METHOD: storeProps.register</a></li>
+                </ol>
+              </li>
+            </ol>
+          <li><a href="#createStore-actionStore">Return - actionStore - OBJECT</a></li>
```

Comment:
Don't need a link for this can remove (since it's not important information for the reader.